plugin/tests/test-extractor.php

```
@@ -0,0 +1,369 @@
+        
+        // Test: Duplicate Prevention
+        echo "\nTest: Duplicate Prevention\n";
+        echo "---------------------------\n";
+        // Reset singleton for duplicate test
+        $reflection = new ReflectionClass('WebComponentsExtractor');
+        $instance = $reflection->getProperty('instance');
+        $instance->setAccessible(true);
+        $instance->setValue(null, null);
+        
+        $extractor = WebComponentsExtractor::getInstance();
+        $first = $extractor->getScriptTagHtml();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$first'. (Unused Code Rules)

(UnusedLocalVariable)